src/pages/farm/Farm.tsx

```diff
@@ -0,0 +1,159 @@
+import FarmOverview from "../../components/farm-info-card/FarmOverview"
+
+function Farm(): ReactElement | null {
+  const [a4dPoolData, a4dUserShareData] = usePoolData(AXIAL_A4D_POOL_NAME)
+  const [currentModal, setCurrentModal] = useState<string | null>(null)
+  const [filter, setFilter] = useState<PoolTypes | "all" | "outdated">("all")
+  const handleClickMigrate = () => {
+    setCurrentModal("migrate")
+  }
+
+  function getPropsForPool(poolName: PoolName) {
+    return {
+      name: AXIAL_A4D_POOL_NAME,
+      poolData: a4dPoolData,
+      userShareData: a4dUserShareData,
+      poolRoute: "/pools/a4d",
+    }
+  }
+  return (
+    <div className={styles.poolsPage}>
+      <TopMenu activeTab="farm" />
+      <ul className={styles.filters}>
+        {[
+          ["all", "ALL"] as const,
+          [PoolTypes.BTC, "BTC"] as const,
+          [PoolTypes.ETH, "ETH"] as const,
+          [PoolTypes.USD, "USD"] as const,
+        ].map(([filterKey, text]) => (
+          <li
+            key={filterKey}
+            className={classNames(styles.filterTab, {
+              [styles.selected]: filter === filterKey,
+            })}
+            onClick={(): void => setFilter(filterKey)}
+          >
+            {text}
+          </li>
+        ))}
+      </ul>
+      <div className={styles.content}>
+        {Object.values(POOLS_MAP)
+          .filter(
+            ({ type, migration, isOutdated }) =>
+              filter === "all" ||
+              type === filter ||
+              (filter === "outdated" && (migration || isOutdated)),
+          )
+          .map(
+            ({ name, migration, isOutdated }) =>
+              [getPropsForPool(name), migration, isOutdated] as const,
+          )
+          .sort(
+            ([a, aMigration, aIsOutdated], [b, bMigration, bIsOutdated]) => {
+              // 1. active pools
+              // 2. user pools
+              // 3. higher TVL pools
+              if (aMigration || bMigration || aIsOutdated || bIsOutdated) {
+                return aMigration || aIsOutdated ? 1 : -1
+              } else if (
+                (a.userShareData?.usdBalance || Zero).gt(Zero) ||
+                (b.userShareData?.usdBalance || Zero).gt(Zero)
+              ) {
+                return (a.userShareData?.usdBalance || Zero).gt(
+                  b.userShareData?.usdBalance || Zero,
+                )
+                  ? -1
+                  : 1
+              } else {
+                return (a.poolData?.reserve || Zero).gt(
+                  b.poolData?.reserve || Zero,
+                )
+                  ? -1
+                  : 1
+              }
+            },
+          )
+          .map(([poolProps, migrationPool]) => (
+            <FarmOverview
+              key={poolProps.name}
+              {...poolProps}
+              onClickMigrate={
+                migrationPool ? () => handleClickMigrate() : undefined
+              }
+            />
+          ))}
+        {/* <FarmOverview
```

Comment:
Avoid pushing commented out code, if you do because you have a reason, you need to add a description for why is commented out.